README.md

```diff
@@ -48,22 +48,20 @@ Install all dependencies:
 yarn
 ```
 
-Build nebula.js visualization:
+Build a nebula.js visualization as a Qlik Sense extension:
 
 ```sh
 yarn build
 ```
 
-Build a nebula visualization as a Qlik Sense extension:
-
-```sh
-yarn sense
-```
-
 |                          [Saas Edition of Qlik Sense]                           |                     [Qlik Sense Enterprise]                      |                            [Qlik Sense Desktop]                            |
 | :-----------------------------------------------------------------------------: | :--------------------------------------------------------------: | :------------------------------------------------------------------------: |
 | Copy sn-table-ext into https://your-tenant.us.qlikcloud.com/console/extensions/ | Copy sn-table-ext into Qlik Management Console (QMC)->Extensions | Copy sn-table-ext into ..\Users\<UserName>\Documents\Qlik\Sense\Extensions |
 
+### Run a sn-table extension against the Saas Edition of Qlik Sense
+
+After building a nebula.js visualization, you can also run the sn-table extension against the Saas Edition of Qlik Sense in the local nebula web development server by adding your webIntegrationId and pointing the engine URL to your tenant following [the bottom of the page](https://qlik.dev/libraries-and-tools/nebulajs/nebula-serve).
```

Comment:
I don't understand what this means. Is there stuff missing on the qlik.dev page? in any case it should not say bottom of the page, stuff could be added at the bottom. You can link to a specific heading instead

Reply:
Sure

Reply:
I still don't get it. Are you saying you can run the nebula dev env against a tenant?

Reply:
I found that nebula has the explanation.
so there are two places to add webIntegrationId in the URL or in the nebula server. 
I am trying to describe them in some sentences. Any suggestions?

 <img width="853" alt="Screenshot 2022-03-09 at 09 38 17" src="https://user-images.githubusercontent.com/4471489/157404028-564b2898-72d7-459e-8801-e24619f8fe1c.png">

Reply:
How about now?

Reply:
So this is about developing? Then it should be in the contribution file. The readme should just include how to build and use the table

Reply:
why should the readme not include how to develop?

Reply:
because we moved it to the contribute file

Reply:
I think only contributors who are interested in and will look at the git contribution file. The majority of people just want to have a try, right?

Reply:
yes, and then they should build the extension and use it somewhere, not start the dev server

Reply:
We could explain more under the Contribution heading what info you can find there

Reply:
aha, good idea. I gonna do the update.

Reply:
And also I will move those into git Contribution file

Reply:
Done